### ISSUE TITLE
How to remove blank line diff?

### ISSUE THREAD

**Issue by AndrejMitrovic** [jeffkaufman/icdiff#173]
Is it possible to remove these big green blocks? I think they're showing blank lines.

<img width="73" alt="Screen Shot 2020-07-06 at 14 54 21" src="https://user-images.githubusercontent.com/15028641/86560148-a4db6f80-bf98-11ea-9ea5-2c2d6dc14629.png">

Might be related to https://github.com/jeffkaufman/icdiff/issues/170.

**Comment by jeffkaufman** [comment]
Could you share an example pair of files that gives this effect, and say what you'd like to see instead?

**Comment by AndrejMitrovic** [comment]
Sure!

For example, test1.cpp:

```cpp
void main ()
{
    int x;
    int y;
}
```

test2.cpp:

```cpp
void main ()
{
    int x;
    int y;
}

void foo ()
{

}
```

Produces:

<img width="827" alt="Screen Shot 2020-07-07 at 10 19 21" src="https://user-images.githubusercontent.com/15028641/86682330-6d71cf00-c03b-11ea-813c-35102b90e01b.png">

I don't need the blank lines to be highlighted with the green rectangles. It's distracting from the rest of the diff. Is there a way to disable highlighting whitespace changes, or just blank line changes?

**Comment by johnowens99** [comment]
Any fix related to this? I'm wondering...

**Comment by jeffkaufman** [comment]
I think this should be fixed with 9dbfbb91d49afffea324d3b1bffd45b24d34a43c and a new `--show-no-spaces` argument. If this looks good, I can make this behavior the default?

**Comment by johnowens99** [comment]
Ah, sorry. I thought this was an open forum. I wasn't sure if there was a way to do this in Github in general.
